finalProject/fizzBuzz.js

```diff
@@ -0,0 +1,6 @@
+// FizzBuzz
+
+for (var i = 0; i <= 100; i++) {
```

Comment:
This is very advanced syntax not taught in Mod0. I'd encourage you to start with basics to make sure you have a solid foundation. If I saw this code in a project, I'd expect you could explain ternary statements pretty well.